pkg/ingester/metrics.go

```diff
@@ -546,6 +547,10 @@ func newTSDBMetrics(r prometheus.Registerer, logger log.Logger) *tsdbMetrics {
 			"cortex_ingester_tsdb_sample_out_of_order_delta_seconds",
 			"Delta in seconds by which a sample is considered out-of-order.",
 			nil, nil),
+		tsdbMaxTime: prometheus.NewDesc(
+			"cortex_ingester_tsdb_head_max_time_seconds",
```

Comment:
I suggest "timestamp" instead of "time" to keep consistency with other metrics exposing a unix timestamp.

```suggestion
			"cortex_ingester_tsdb_head_max_timestamp_seconds",
```

Reply:
I would keep it consistent with original metric `prometheus_tsdb_head_max_time_seconds`. WDYT?